packages/backend/src/assets/ai.json

```diff
@@ -29,6 +31,8 @@
       "recommended": [
+        "hf.bartowski.granite-3.1-8b-instruct-GGUF",
+        "hf.lmstudio-community.granite-3.0-8b-instruct-GGUF",
```

Comment:
question: what is the difference with existing row 38 ?

Reply:
Ordering

Reply:
![image](https://github.com/user-attachments/assets/62822146-c689-4fc0-9135-74d9f012d7d3)

issue: to me it looks like a duplicated entry ?

Reply:
Fixed